lib/components/tabs.vue

```diff
@@ -146,28 +142,28 @@
              * Move to next tab
              */
             nextTab() {
-                this.setTab(this.currentTab, false, 1);
+                this.setTab(this.currentTab + 1, false, 1);
             },
 
             /**
              * Move to previous tab
              */
             previousTab() {
-                this.setTab(this.currentTab, false, -1);
+                this.setTab(this.currentTab - 1, false, -1);
```

Comment:
Also, this removes the new for people to call previous and next methods directly during external control.  They can now just increment the value and it will look for the next tab if available.,